patch/0001-fix-driver-optoe-twoaddr-a2h-read-write-bug.patch

```diff
@@ -0,0 +1,78 @@
+From 4338f175d03e83b3dd187c4d2af193e9e6f86039 Mon Sep 17 00:00:00 2001
+From: "Song,Qinghua" <songqh@clounix.com>
+Date: Fri, 20 Sep 2024 19:32:52 +0800
+Subject: [PATCH] fix driver optoe twoaddr a2h read/write bug
+
+---
+ drivers/misc/eeprom/optoe.c | 29 ++++++++++++++++++++++++++---
+ 1 file changed, 26 insertions(+), 3 deletions(-)
+
+diff --git a/drivers/misc/eeprom/optoe.c b/drivers/misc/eeprom/optoe.c
+index 16287fdc5..c6fdce9c9 100644
+--- a/drivers/misc/eeprom/optoe.c
++++ b/drivers/misc/eeprom/optoe.c
+@@ -284,7 +284,6 @@ static uint8_t optoe_translate_offset(struct optoe_data *optoe,
+ 	if (optoe->dev_class == TWO_ADDR) {
+ 		if (*offset > 255) {
+ 			/* like QSFP, but shifted to client[1] */
+-			*client = optoe->client[1];
+ 			*offset -= 256;
+ 		}
+ 	}
+@@ -304,6 +303,27 @@ static uint8_t optoe_translate_offset(struct optoe_data *optoe,
+ 
+ 	return page;  /* note also returning client and offset */
+ }
++static void optoe_update_client(struct optoe_data *optoe, loff_t *offset, struct i2c_client **client)
```

Comment:
Please use the Linux kernel coding style. Mainly please use tabs.

Reply:
Have used the Linux kernel coding style-use tabs，please help to check again，thank you！